### PR TITLE
Dynamic Profile URL with new UI

### PR DESCRIPTION
**Pull request by omokdad**
I deleted my migrations, you don't have to

### CODE REVIEW

File: social/author/models.py [hunk 1]
```diff
@@ -26,12 +26,13 @@ class Author(models.Model):
...
     gender = models.CharField(max_length=1, choices=genderChoices, blank=True)
     gitURL = models.TextField(blank=True)
+    #uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
 
     def __str__(self):
         return str(self.id)
 
     def getAuthorURL(self):
-        return settings.LOCAL_HOST + 'a/' + self.id
+        return settings.LOCAL_HOST + 'author/' + self.uuid
```
Comment (emmettu) [inline social/author/models.py]: should be self.id?

File: social/author/templates/author/edit.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
               <h3 class="box-title"> Edit {{request.user.username}}</h3>
             </div>
             <div class="box-body">
-                <form action="/a/edit_post/" method="POST">
+                <form action="/author/edit_post/" method="POST">
```
Comment (emmettu) [inline social/author/templates/author/edit.html:23]: in the future we should use Jinja urls

File: social/author/templates/author/index.html [hunk 3]
```diff
@@ -20,7 +19,8 @@
               <h3 class="box-title"> Add a Post</h3>
             </div>
             <div class="box-body">
-			    <form action="/a/author_post/" method="POST">
+			    <form action="/author/author_post/" method="POST">
+
```
Comment (emmettu) [inline social/author/templates/author/index.html:23]: same comment, jinja urls

File: social/social/templates/includes/navigation.html [hunk 4]
```diff
@@ -32,15 +32,15 @@
           <img src="/static/img/user2-160x160.jpg" class="img-circle" alt="User Image">
         </div>
         <div class="pull-left info">
-          <p><a href="/a/profile/">{{request.user.username}}</a></p>
-          <a href="/a/edit/"> Edit Profile</a>
+          <p><a href={% url 'author:profile' request.user.id %}>{{request.user.username}}</a></p>
```
Comment (emmettu) [inline social/social/templates/includes/navigation.html:35]: nice

**Review by emmettu — APPROVED** [review]

File: social/social/templates/includes/navigation.html (hunk 4 above)
Comment (k----n) [reply social/social/templates/includes/navigation.html:35]: yesss namespace

**Pull request merged**